Dockerfile

```diff
@@ -41,5 +41,9 @@ RUN dnf install -y dnf-plugins-core \
     && dnf install -y fence-agents-all fence-agents-aws fence-agents-azure-arm fence-agents-gce \
     && dnf clean all -y
 
+# Store the avavliable agents names (fence_*) into readonly fence_agents_list file
+RUN ls /usr/sbin/fence* > fence_agents_list \
```

Comment:
fence_agents_list should probably contain a list of the names of the  fence agents and not list of paths of the fence agent

Reply:
Makes sense :)